pkg/mqtt5/client.go

```diff
@@ -28,6 +28,8 @@ const (
 )
 
 type Mqtt5Client struct {
```

Comment:
create an interface for the future unit tests

Reply:
added